src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,23 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany(cascade = CascadeType.DETACH, fetch = FetchType.LAZY)
```

Comment:
1. what is the default fetch type for @OneToMany?
2. why have you decided to use this cascade type? what do you want to achieve with it?

Reply:
1. Not sure what fetch type is in general and what is the purpose of it. 
2. It was random as I have no understanding what cascade type to use in what case. I tried all of them